examples/hotelling_law/hotelling_law/agents.py

```diff
@@ -165,8 +162,8 @@ def __init__(self, model):
         self.preferred_store = None
 
     def determine_preferred_store(self):
-        consumer_preference = self.model.consumer_preferences
-        stores = self.model.get_store_agents()
+        consumer_preference = self.model.agents_by_type[ConsumerAgent]
```

Comment:
Is this correct? Why should is now be an AgentSet, when it first was a string (`model.consumer_preferences="default",`)

Reply:
this is indeed probably a mistake on my end